config.py

```diff
@@ -1987,6 +2007,22 @@ def gen_log_kwargs(
            f'are: "FLASH", "T1", and "auto".')
     raise ValueError(msg)
 
+if contrasts:
+    for contrast in contrasts:
+        if isinstance(contrast, tuple):
+            if len(contrast) != 2:
+                raise ValueError("Contrasts' tuples MUST be two conditions")
+        else:
+            if "name" not in contrast.keys():
+                raise ValueError("One contrast is missing a name")
+            if "conditions" not in contrast.keys():
+                raise ValueError("One contrast is missing a condition list")
+            if "weights" not in contrast.keys():
+                raise ValueError("One contrast is missing a weight list")
+            if len(contrast["conditions"]) != len(contrast["weights"]):
+                raise ValueError(f"Contrast {contrast['name']} has an"
+                                 f"inconsistent number of conditions/weights")
```

Comment:
I don’t think validation should be done here. It needs to be done also when you have a custom config file so after the first import of config